docs/helm-chart/keda.rst

```diff
@@ -0,0 +1,72 @@
+ .. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+Autoscaling with KEDA
+---------------------
+
+*This feature is still experimental.*
+
+KEDA stands for Kubernetes Event Driven Autoscaling.
+`KEDA <https://github.com/kedacore/keda>`__ is a custom controller that
+allows users to create custom bindings to the Kubernetes `Horizontal Pod
+Autoscaler <https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/>`__.
+We have built a scaler that allows users to create scalers based on
```

Comment:
```suggestion
We have built a scalers that allows users to create scalers based on
```

Reply:
There are 3 scalers that can be used with Airflow and all build by Daniel or me. But I assume we should mention only PostgreSQL later on as Postgres is part fo chart.

Reply:
we should describe the scaller for MySQL as well, but I would prefer to do it as a separate ticket. for now, I am changing the structure, but without changing the content.

Helm Chart supports PostgreSQL and MySQL.  in cluster postgres is only intended for testing.

Reply:
`a scaler` or `scallers` (but not `a scallers`)